src/cellRenderers/renderCheckbox.tsx

```diff
@@ -19,7 +19,12 @@ const checkboxLabelClassname = `rdg-checkbox-label ${checkboxLabel}`;
 
 const checkboxInput = css`
   @layer rdg.CheckboxInput {
-    all: unset;
```

Comment:
When the checkbox has `all: unset` style then for some reason `onChange` is not getting called on shift click